src/delete.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-cycle */
+import { saveTasks } from './localStorage';
+import addTasks from './showTasks';
+import { tasks } from './taskClass';
+
+const deleteTask = () => {
+  const clearBtn = document.querySelector('#clear');
+  clearBtn.addEventListener('click', () => {
+    const result = tasks.filter((x) => x.completed === false);
+    tasks.splice(0);
+    tasks.push(...result);
+    saveTasks();
+    addTasks();
+  });
+};
+
+export default deleteTask;
```

Comment:
- [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. Currently when I delete a task let us say `index:0` it doesn't update the remaining index so that an item which was on `index:1` take `index:0` please work on that
- [ ] When I clear all completed tasks it doesn't update index from the local storage as well. Kindly work on that